models/xero__invoice_line_items.sql

```diff
@@ -18,7 +23,17 @@ with line_items as (
     select *
     from {{ var('contact') }}
 
-), joined as (
+)
+
+{% if using_tracking_categories %} 
+, pivoted_tracking_categories as (
+
+    select *
+    from {{ ref('int_xero__invoice_line_item_pivoted_tracking_categories') }}
+
+){% endif %}
+
+, joined as (
```

Comment:
This hanging parenthesis is an anti-pattern from how we usually handle this situation and is a bit confusing to understand/maintain since the ending parenthesis is hiding by the endif. We should have it instead be something along the following.

```suggestion

{% if using_tracking_categories %} 
) , pivoted_tracking_categories as (

    select *
    from {{ ref('int_xero__invoice_line_item_pivoted_tracking_categories') }}

{% endif %}

), joined as (
```

Reply:
Updated.